README.md

```diff
@@ -78,6 +78,7 @@ We recommend setting up a Conda environment for development:
 ```bash
 conda create --name koalas-dev-env python=3.6
 conda activate koalas-dev-env
+conda install -c conda-forge pyspark=2.4
```

Comment:
This was missed https://github.com/databricks/koalas/commit/4585dfce4ef06b6af645058ce3acd2473cf7840c#diff-04c6e90faac2675aa89e2176d2eec7d8L95.

`requirements-dev.txt` doesn;t have pyspark dependency because it can be installed via downloading the release as well.